README.md

```diff
@@ -1,197 +1,212 @@
 
-Instructions for importing the environment JSON is located [here](https://thinkster.io/tutorials/testing-backend-apis-with-postman/managing-environments-in-postman). The three environment files are:
+**Setting up environment variables**
 
-- Fhir_Local_Env.json
-- Fhir_Dev_Env.json
-- Fhir_Prod_Env.json
+Set up the following three environment files:
 
-The variables required in the POSTMAN collection can be found in `Info_Output.yml` or by running `serverless info --verbose`
++ `Fhir_Local_Env.json`
++	`Fhir_Dev_Env.json`
++	`Fhir_Prod_Env.json`
 
-- API_URL: from Service Information:endpoints: ANY
-- API_KEY: from Service Information: api keys: developer-key
-- CLIENT_ID: from Stack Outputs: UserPoolAppClientId
-- AUTH_URL: `https://<CLIENT_ID>.auth.<REGION>.amazoncognito.com/oauth2/authorize`
+For instructions on importing the environment JSON, click [here](https://thinkster.io/tutorials/testing-backend-apis-with-postman/managing-environments-in-postman).
 
-To know what all this FHIR API supports please use the `GET Metadata` postman to generate a [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
+The following variables required in the Postman collection can be found in `Info_Output.yml` or by running `serverless info --verbose`:
 
-### Authorizing a user
++	`API_URL: from Service Information:endpoints: ANY`
++	`API_KEY: from Service Information: api keys: developer-key`
++ `CLIENT_ID: from Stack Outputs: UserPoolAppClientId`
++ `AUTH_URL: https://<CLIENT_ID>.auth.<REGION>.amazoncognito.com/oauth2/authorize`
 
-FHIR Works on AWS solution uses role based access control (RBAC) to determine what operations and what resource types the requesting user has access too. The default ruleset can be found here: [RBACRules.ts](src/RBACRules.ts). For users to access the API they must use an OAuth access token. This access token must include scopes of either:
+To find what the FHIR Server supports, use the `GET Metadata` postman to generate a [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
 
-- `openid profile` Must have both
-- `aws.cognito.signin.user.admin`
+**Authorizing a user**
 
-Using either of the above scopes will include the user groups in the access token.
+FHIR Works on AWS solution uses Role-Based Access Control (RBAC) to determine what operations and what resource types the requesting user has access too. The default ruleset can be found in [RBACRules.ts](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/src/RBACRules.ts). To access the API, you must use the OAuth access token. This access token must include scopes of either `openid`, `profile` or `aws.cognito.signin.user.admin`. 
```

Comment:
```suggestion
FHIR Works on AWS uses Role-Based Access Control (RBAC) to determine what operations and what resource types the requesting user has access to. The default rule set can be found in [RBACRules.ts](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/src/RBACRules.ts). To access the API, you must use the OAuth access token. This access token must include scopes of either `openid`, `profile` or `aws.cognito.signin.user.admin`. 
```

Reply:
Changed this line from " FHIR Works on AWS uses Role-Based Access Control (RBAC) to determine what operations and what resource types the requesting user has access to.

to "FHIR Works on AWS uses Role-Based Access Control (RBAC) to determine what operations and what resource types a user can access."